buildSrc/src/main/java/Publication.kt

```diff
@@ -3,11 +3,7 @@ import org.gradle.api.distribution.DistributionContainer
 import org.gradle.api.file.CopySpec
 import java.io.File
 
-private object Consts {
-    val taskRegex = Regex("(.*)DistZip")
-}
```

Comment:
dead code, suprised detekt doesn't detect this

Reply:
doesnt _detekt_ 😜